### PR TITLE
fixed OKAPI GPX download; fixes #1270

### LINKED ISSUE #1270: OKAPI GPX download is broken
The OKAPI GPX download of search results no longer works. On the OC.PL website I get an empty page, and on my develsite the error message:

Fatal error: Class 'okapi\Facade' not found in /mnt/ocpl/lib/okapi_gpx.php on line 16

### PR DESCRIPTION
**Pull request by following5**
This is part 1 of the bugfix. Part 2 is aready deployed as 5577d29e.

### CODE REVIEW

File: okapi_settings.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ function get_okapi_settings()
         'IMAGE_MAX_UPLOAD_SIZE' => $config['limits']['image']['filesize'] * 1024 * 1024,
         'IMAGE_MAX_PIXEL_COUNT' => $config['limits']['image']['height'] * $config['limits']['image']['width'],
         'OC_NODE_ID' => $oc_nodeid,
-        'OC_COOKIE_NAME' => $config['cookie']['name'],
+        'OC_COOKIE_NAME' => $config['cookie']['name'].'_auth',
```
Comment (kojoty) [inline okapi_settings.php]: @following5, let me understand how OKAPI uses cookie?  Is it necessary for authorization? If yes - I'm afraid it's not enough, if not this change is not necessary...

Comment (following5) [reply okapi_settings.php]: Okapi needs to know the session ID of the logged-in user. Then it queries the user ID from the `sys_session` table.

I have tested this change and it worked. :-)

Comment (following5) [reply okapi_settings.php]: Please see 5577d29e, that is the corresponding Okapi-side change.

Comment (kojoty) [reply okapi_settings.php]: Got it - now everything is clear - I've missed that OKAPI needs user sessionId here... Thanks for this fix - is it ready to merge?

Comment (following5) [reply okapi_settings.php]: yep

**Pull request merged**